extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -0,0 +1,141 @@
+# KHR_draco_mesh_compression
+
+## Contributors
+
+* Fan Zhang, Google, <mailto:zhafang@google.com>
+* Ondrej Stava, Google, <mailto:ostava@google.com>
+* Frank Galligan, Google, <mailto:fgalligan@google.com>
+* Kai Ninomiya, Google, <mailto:kainino@google.com>
+* Patrick Cozzi, Cesium, [@pjcozzi](https://twitter.com/pjcozzi)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a schema to use [Draco geometry compression](https://github.com/google/draco) libraries in glTF format. This allows glTF to support streaming compressed geometry data instead of the raw data.
+
+The [conformance](#conformance) section specifies what an implementation must do when encountering this extension, and how the extension interacts with the attributes defined in the base specification.
+
+## glTF Schema Updates
+
+Draco geometry compression library could be used for `primitive` by adding an `extension` property to a primitive, and defining its `KHR_draco_mesh_compression` property.
+
+The following picture shows the structure of the schema update. 
+
+**Figure 1**: Structure of geometry compression extension.
+![](figures/structure.png)
+
+In general, we will use the extension to point to the buffer that contains the compressed data. The major change is that the `accessor` in an extended `primitive` no
+longer points to a `bufferView`. Instead, the `attributes` of a primitive will use the decompressed data. This is valid because in glTF 2.0, `bufferView` is not required in `accessor`, although if it is not present, it will be used with `sparse` field to act as a sparse accessor. In this extension, we will ignore the `bufferView` property.
+
+Usage of the extension must be listed in the `extensionUsed` and `extensionsRequired`. 
```

Comment:
This looks a good solution. Just two thoughts. One thing is that a loader should determine whether or not support the extension when checking `extensionUsed`, if not then download the binary source with uncompressed assets, otherwise download the one with compressed assets. And throughout the loading, it should process the extension consistently, either use it or omit it.

I think the fallback has one problem left that if the loader supports the extension but there's some attributes, e.g. user defined generic attributes, that are not compressed in the extension. In this case, that attribute will have conflict `accessor` since the `bufferView` should be different because they point to different binary file. It could not use data in compressed binary source at all, but it needs to when the extension is available.